test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -214,8 +220,6 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
     }
 
     Context "Test Unicode escape sequences." {
-        # These tests require the file to be saved with a BOM.  Unfortunately when this UTF8 file is read by
```

Comment:
why was this removed?

Reply:
This comment is no longer accurate. PowerShell no longer relies on BOM to process anything, and all files are processed as UTF8 unless some other encoding is specified with the appropriate cmdlet.